index.html

```diff
@@ -2,11 +2,56 @@
 <html lang="ru">
   <head>
     <meta charset="utf-8">
-    <title>HTML Academy: Седона</title>
+    <title>Седона</title>
   </head>
   <body>
+    <header class="main-header">
+      <nav class="main-navigation">
+        <ul class="site-navigation">
+          <li><a href="info.html">Информация</a></li>
+          <li><a href="photo-video.html">Фото и видео</a></li>
+          <li><a href="map.html">Карта штата</a></li>
+          <li><a href="hotels.html">Гостиницы</a></li>
+        </ul>
+      </nav>
+      <a class="main-header-logo">
+      <img src="img/index.logo.svg" width="100" height="100" alt="Логотип">
+      </a>
+    </header>
+    <main class="conteiner">
```

Comment:
Обрати внимание на наименование классов
```suggestion
    <main class="container">
```